src/main/groovy/com/github/dispader/manifesto/ManifestoPlugin.groovy

```diff
@@ -84,9 +72,6 @@ class ManifestoPlugin implements Plugin<Project> {
                     }
 
                     attributes('Implementation-Timestamp': new Date())
-
-                    if ( it instanceof Jar ) { jar_configured = true }
```

Comment:
Hrm— I don't understand the commit comment, here.  Lines 88 and 89 (pre-commit) set the static class attribute to `true`.

This code is absolutely horrible, I know— but it was added because I was debugging the code, and noticed that the code being run to set the manifest version was being run many, many times during a build.  This was put in to try and prevent that.

All that does leave the question as to "why is this code being executed multiple times in the first place?," and "why do we run all this code at the time the plugin is added?"